apps/src/applab/applab.js

```diff
@@ -1121,6 +1121,10 @@ Applab.execute = function() {
       var session = studioApp.editor.aceEditor.getSession();
       annotationList.attachToSession(session, studioApp.editor);
       annotationList.clearRuntimeAnnotations();
+      studioApp.editor.aceEditor.session.on("change", function () {
```

Comment:
Oh right, because the underlying Ace editor is updated.  Worth validating, but I think you're right.  Does this behavior need a test?